drone/exec.go

```diff
@@ -141,6 +149,7 @@ func execCmd(c *cli.Context) error {
 
 	var exits []int
 
+axes:
```

Comment:
edit: nervermind was looking at the wrong revision. I see where it is being used